bindings/node/binding_test.js

```diff
@@ -0,0 +1,9 @@
+/// <reference types="node" />
```

Comment:
How do these new tests in `bindings` work? They don't seem to be getting run in CI.

I would rather put these new tests in a separate PR if possible, along with a check in CI via the root justfile.

Reply:
This is something created by `tree-sitter` cli with the latest version, I will try to see where this is created

Reply:
Oh, interesting. No worries then, it can stick around if it's the same version used in CI (which should probably be bumped)

Reply:
I'm actually unable to produce the `bindings_test.js`, I've deleted the file :\